documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/CertificationScheme/Certification_Roles_and_Responsibilities.md

```diff
@@ -43,52 +43,68 @@ The IDSA is not actively involved in a participant or core component certificati
+-   Approval of the Evaluation Facilites. 
+-   - For the Assurance Level 2, Evaluation Facilites are needed. They evaluate the components in detail and make sure that they act according to the IDS Certification Scheme. 
+-   - Before they start evaluating, the Certification Body ensures, that the evaluation facility's tools and tests are valid. This is beeing done within the approval process. During dedicated workshops the Certification Body examinates the skills of the evaluators. 
 
--   Analyzing existing "base" certificates (e.g. for organizations or for software and hardware security components) to determine their validity and sufficiency, and deciding about their acceptance within the Industrial Data Space certification scheme.
+-   Monitoring the performance of the Evaluation Facilites.
+-   - To ensure highest quality of the certification process the Certification Body participates in the evaluations on a selective basis.
+-   - The Certification Body also reminds the Evaluation Facility to get re-approved after two years if no evaluation has been conducted during that period.
+-   - Whenever new guidelines are developed and published by the association, the Certification Body informs the Evaluation Facilities. 
+-   -   Decision about approval or exclusion of Evaluation Facilities for/from executing IDS evaluations (based on ongoing monitoring and [CRIT-EF]).
 
--   Reviewing and commenting on the evaluation reports received from Evaluation Facilities.
+- Issuing the certificates to the applicants. 
+- - Reviewing and commenting on the evaluation reports received from Evaluation Facilities.
+-   - Making final decision about the award or denial of a certificate and publishing the awarded certificates.
+-   - Keeping track of the certification processes currently in progress and communicating them transparently at the IDSA website.
+-   The Certification Body only grants the certificate (called evaluation certificate subsequently) only if both the Evaluation Facility and the experts of the Certification Body have come to the conclusion that all preconditions are fulfilled. The certification body sends the certificate directly to the applicant, at the same time informing the Evaluation Facility of its decision. 
 
--   Approval of applications for certification.
+-   Guiding applicants through the certification process.
+-   - The Certification provides and maintans the list of approved Evaluation Facilites on the IDSA website.
+-   - By contacting supportoffice@internationaldataspaces.org applicants can reach out for support on the whole certification process. 
 
--   Making final decision about the award or denial of a certificate and publishing the awarded certificates.
+-   Ongoing monitoring of certification-relevant external developments (e.g. new attack patterns which might circumvent certified security measures).
```

Comment:
I really like those additional points below :+1:
Can we find either a new heading for them or integrate them into one of the existing points, e.g. the first aspect of "Maintaining the IDS Certification Scheme:"?